superset-frontend/src/views/CRUD/utils.tsx

```diff
@@ -45,13 +47,14 @@ const createFetchResourceMethod = (method: string) => (
     const { json = {} } = await SupersetClient.get({
       endpoint: `${resourceEndpoint}?q=${queryParams}`,
     });
-
-    return json?.result?.map(
+    const data = json?.result?.map(
       ({ text: label, value }: { text: string; value: any }) => ({
         label,
         value,
       }),
     );
+
+    return options.concat(data);
```

Comment:
I believe some of these results are paginated so each request will end up with multiple `me` entries.

Reply:
Actually, nvm. Looks like these results are merged in with the existing ones and get properly deduped. 👍